samples/snippets/src/main/java/com/example/spanner/CustomTimeoutAndRetrySettingsExample.java

```diff
@@ -53,14 +53,23 @@ static void executeSqlWithCustomTimeoutAndRetrySettings(
         .setRetrySettings(
             RetrySettings.newBuilder()
                 // Configure retry delay settings.
+                // The initial amount of time to wait before retrying the request.
                 .setInitialRetryDelay(Duration.ofMillis(500))
-                .setMaxRetryDelay(Duration.ofSeconds(64))
+                // The maximum amount of time to wait before retrying. I.e. after this value is
+                // reached, the wait time will not increase further by the multiplier.
+                .setMaxRetryDelay(Duration.ofSeconds(10))
+                // The previous wait time is multiplied by this multiplier to come up with the next
+                // wait time, until the max is reached.
                 .setRetryDelayMultiplier(1.5)
 
                 // Configure RPC and total timeout settings.
-                .setInitialRpcTimeout(Duration.ofSeconds(60))
-                .setMaxRpcTimeout(Duration.ofSeconds(60))
-                .setRpcTimeoutMultiplier(1.0)
+                // Timeout for the first RPC call. Subsequent retries will be based off this value.
```

Comment:
In the [doc](https://cloud.google.com/spanner/docs/custom-timeout-and-retry#java): 

> In the following sample, a timeout of 60 seconds is set for the given operation. If the operation takes longer than this timeout, a DEADLINE_EXCEEDED error is returned.

If we update the code sample to this, I don't it match this statement. We probably should update nodejs code sample to set all values to 60s.

Reply:
Oh, good catch. I'll change that back, and I'll also open a PR for the Node sample to be in line with the documentation page.